util/Migrator/DbScripts/2021-04-13_00_CipherPasswordPrompt.sql

```diff
@@ -0,0 +1,202 @@
+﻿IF COL_LENGTH('[dbo].[Cipher]', 'Reprompt') IS NULL
+BEGIN
+    ALTER TABLE [dbo].[Cipher]
+        ADD [Reprompt] TINYINT NULL;
+END
+GO
+
+UPDATE
+    [dbo].[Cipher]
+SET
+    [Reprompt] = 0
```

Comment:
This script, if the `NOT NULL` column remains, would need to have a where clause added,
```sql
WHERE
    [Reprompt] IS NULL
```